index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Nathan Explosion</title>
+		<link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+		<nav>
+			<a href="index.html">Bio</a>
+			<a href="Experience.html">Experience</a>
+			
+		</nav>
+	</head>
+ <body>
+	<header>Nathan Explosion</header>
+	
+	<p><header id='head2'>Bio</header></p>
+		<p>Young Nathan, moments before a truck crashes into his classroom
+		Nathan was raised in New Port Richey, Florida, the sole child of Oscar, implied to have been in the military, and Rose Explosion. Nathan was a very quiet child to the point of being regarded as mute, and did not speak until the age of five. In Dethgov, when his classmates and teacher were killed after he won the election for class president in the second grade, Nathan remained virtually unaffected. He was an unremarkable student in high school, excelling only in frog dissection and football. He eventually dropped out without earning his diploma (though he tried to earn his GED years later).</p>
+
+	<img src="images/nathan.png">
```

Comment:
This image is not loading because you are requesting `images/nathan.png`, however the path is `image/Nathan.png`. Try to name your file in a consistent way to avoid these situations. 🔨